### PR TITLE
Add Facebook metadata

### LINKED ISSUE #575: Add Facebook metadata
**Is your feature request related to a problem? Please describe.**
Our article page is currently missing metadata. This PR will add all the available Facebook meta.

**Describe the solution you'd like**
Add the following fields, including the logic necessary to render them correctly:
- `fb:app_id` - `1609039196070050` [OR ARES]
- `fb:admin` - `100004154058350` [OR ARES]

Pass in as variables, to live in a metadata config file here in lib: `https://github.com/bbc/simorgh/tree/latest/src/app/lib`. 

**Additional context**
Some fields might require further discussion, and we will initially hard-code them.

**Testing notes**
- Unit tests are adequate

### PR DESCRIPTION
**Pull request by jtart**
Resolves #575

_Adds Facebook metadata and tests._

- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/components/Metadata/index.jsx [hunk 1]
```diff
@@ -44,6 +46,8 @@ const Metadata = ({
       {metaTags.map(tag => (
         <meta name="article:tag" content={tag} key={tag} />
       ))}
+      <meta name="fb:admin" content={facebookAdmin} />
```
Comment (pjlee11) [inline src/app/components/Metadata/index.jsx]: This should be `name="fb:admins"`

**Review by pjlee11 — CHANGES_REQUESTED** [review]

**Review by pjlee11 — DISMISSED** [review]
Nice work

**Review by sareh — DISMISSED** [review]
👍 Looks good!

**Review by pjlee11 — DISMISSED** [review]

**Review by sareh — DISMISSED** [review]
👍 Looks good.

**Review by pjlee11 — APPROVED** [review]

**Review by sareh — APPROVED** [review]

**Pull request merged**